va/validation-authority_test.go

```diff
@@ -273,6 +280,25 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertNotError(t, err, chall.Path)
 	test.AssertEquals(t, len(log.GetAllMatching(`redirect from ".*/302" to ".*/301"`)), 1)
 	test.AssertEquals(t, len(log.GetAllMatching(`redirect from ".*/301" to ".*/valid"`)), 1)
+	test.AssertEquals(t, len(log.GetAllMatching(`Resolved addresses for localhost: \[127.0.0.1\]`)), 3)
+
+	log.Clear()
```

Comment:
Please break out these two test cases into separate test functions. We haven't been good about this to do, but I think it's worth cleaning up. You'll probably want to pull out some of the setup code into a helper function in the process.